src/staging/src/components/megaMenu.js

```diff
@@ -1,20 +1,267 @@
+                  </span>
+                </li>
+                <li className="menu-button menu-header">
+                  <Link className="menu-button__title" to={item.url || '/'}>
+                    {item.title}
+                  </Link>
+                </li>
+                {item.strapiChildren.map((page, index) => (
+                  <li key={index}
+                    
+                    className={'menu-button menu-button--' + (selections[page.treeLevel] === page ? "selected" : "unselected")}>
+                      <span className="menu-button__title"
+                        role="button"
+                        tabIndex={0}
+                        onFocus={e => menuFocus(e, page)}
+                        onMouseOver={e => sectionHover(e, page, menuMode)}
+                        onKeyDown={e => {
+                          if ((e.key === 'Enter') || (e.key === ' ')) {
+                            navigatePage(e, page, menuMode)
+                          }
+                        }}
+                        onClick={e => navigatePage(e, page, menuMode)}>
+                          {page.title}
+                          {page.hasChildren && (
+                            <i className="menu-button__arr fa fa-chevron-right"></i>
+                          )}
+                      </span>
+                    </li>
+                  )
+                )}
+              </ul>
+            </nav>
+            
+              {item.strapiChildren.map((page, index) => (
+                  <div key={index} className={'menu-children menu-children-exist--' + page.hasChildren + ' menu-level-' + item.treeLevel +'-children menu-children--'  + (page === selections[page.treeLevel] ? "selected" : "unselected")}>
+                  {generateMenus(page, menuMode)}
+                  </div>            
+                )
+              )}
+            
+            </>
+        )}
+        {(!item.hasChildren) && (
+          <nav>
+            <ul>
+              <li className="menu-button menu-header">
+                <Link className="menu-button__title" to={item.url || '/'}>
+                  {item.title}
+                </Link>
+              </li>
+            </ul>
+          </nav>
+        )}
+      </>
+    )
+  }
 
-const MegaMenu = ({ content }) => {
   return (
-    <div className="nav-container">
-      <div className="d-none d-md-block">
-        <DesktopMegaMenu linkStructure={content} />
-      </div>
-      <div className="d-block d-md-none">
-        <MobileMegaMenu linkStructure={content} />
+    <>
+      <div className={'menu-toggle menu-mode--' + menuMode + ' menu-toggle--' + (isMenuOpen ? "open" : "closed")}
+        role="button"
+        tabIndex={0}
+        onKeyDown={e => {
+          if ((e.key === 'Enter') || (e.key === ' ')) {
+            toggleMenu(e)
+          }
+        }}
+        onClick={e => toggleMenu(e)}>
+        <nav className="menu-open"><i className="fa fa-bars"></i></nav>
```

Comment:
On some screen sizes, this hamburger button seems to be vertically misaligned

Reply:
thanks for the catch, will fix

Reply:
this uncovered a lot of issues with the desktop breakpoint. have switched to bootstrap mixins for breakpoints, way better, suggest adopt across site.